include/mbedtls/ccm.h

```diff
@@ -38,68 +48,71 @@ extern "C" {
 #endif
 
 /**
- * \brief          CCM context structure
+ * \brief   The CCM context-type definition. The CCM context is passed 
+ *			to the APIs called.
  */
 typedef struct {
-    mbedtls_cipher_context_t cipher_ctx;    /*!< cipher context used */
+    mbedtls_cipher_context_t cipher_ctx;    /*!< The cipher context used. */
 }
 mbedtls_ccm_context;
 
 /**
- * \brief           Initialize CCM context (just makes references valid)
- *                  Makes the context ready for mbedtls_ccm_setkey() or
- *                  mbedtls_ccm_free().
+ * \brief           This function initializes the specified CCM context, 
+ *					to make references valid, and prepare the context 
```

Comment:
Please use tabs here.

Reply:
Fixed everywhere in file. Will make sure new files submitted are also aligned with this.